cmd/faFilter/faFilter.go

```diff
@@ -66,6 +70,7 @@ func main() {
 	var end *int = flag.Int("end", -1, "Retains the sequence before this position.")
 	var name *string = flag.String("name", "", "Specifies the fasta record name.")
 	var notName *string = flag.String("notName", "", "Returns all fasta records except for this input.")
+	var nameContains *string = flag.String("nameContains", "", "Returns all fasta records whose name contains this input.")
```

Comment:
maybe include in the usage if the string matching has to be case sensitive or not?

Reply:
Just updated! Thanks!